src/app/register/register.component.html

```diff
@@ -12,9 +12,14 @@
           text="Donor Registration"
         ></biggive-heading>
       </div>
+      @if (redirectPath.startsWith('/regular-giving')) {
+        <p>Please register for a a Big Give donation account to set up your regular giving agreement.</p>
+        <p>You will be able to log-in any time to see your regular or cancel the agreement.</p>
```

Comment:
```suggestion
        <p>You will be able to log-in any time to see your donations or cancel the agreement.</p>
```
I missed that there was another 'your regular'. This change matches the one you committed above.

Reply:
Thanks!